client/index.html

```diff
@@ -98,6 +99,53 @@ <h1 class="title">Sign in to The Lounge</h1>
 								<button type="submit" class="btn">
 									Sign in
 								</button>
+								<% if (signUp) { %>
+								<a href data-target="#sign-up">
+									Sign up
+								</a>
+								<% } %>
+							</div>
+						</div>
+					</form>
+				</div>
+				<div id="sign-up" class="window">
+					<form class="container" method="post" action="">
+						<div class="row">
+							<div class="col-xs-12">
+								<h1 class="title">Sign up to The Lounge</h1>
+							</div>
+							<div class="col-xs-12">
+								<label>
+								Email
+								<input class="input" type="email" name="email" required="required">
+								</label>
+							</div>
+							<div class="col-xs-12">
+								<label>
+								Username
+								<input class="input" name="user" required="required">
+								</label>
+							</div>
+							<div class="col-xs-12">
+								<label class="port">
+								Password
+								<input class="input" type="password" name="password" required="required">
+								</label>
+							</div>
+							<div class="col-xs-12 error" style="display: none;">
+								Sign up failed. May be the username is already used or is invalid.
```

Comment:
`s/May be/maybe/`, or alternatively use something like `This username is already in use or is invalid`